app/controllers/textile_controller.rb

```diff
@@ -52,4 +60,10 @@ def form_params
       [field['key'], field['value']]
     end || []
   end
+
+  # Field-less strings are strings that do not have a field header (#[Field]#).
+  def fieldless_string
+    # Extract all characters before a field header (#[Field]#)
+    @fieldless_string ||= params[:source][/^([\s\S]*?)(?=\n{,2}#\[.+?\]#|\z)/, 1]
```

Comment:
this looks like a field parsing string, that a) is a duplicated that we have defined elsewhere; b) probably shouldn't be in this controller at all.